tests/Jira/ApiTest.php

```diff
@@ -149,6 +149,114 @@ public function testFindVersionByName()
 		);
 	}
 
+	public function testGetResolutions()
```

Comment:
All these tests look almost the same. Proposing to create one universal test with a data provider, that would accept these parameters:

* json filename (e.g. `api_resolution.json`)
* API url (e.g. `/rest/api/2/resolution`)
* expected result (maybe we can make resolutions/fields/statuses/etc. have same IDs and then we don't need to change `$expected` variable at all